tests/oaoo.test.ts

```diff
@@ -130,18 +130,37 @@ describe("oaoo-tests", () => {
       await wfCtxt.sleep(durationSec);
       return;
     }
+
+    @Workflow()
+    static async recvWorkflow(wfCtxt: WorkflowContext, timeoutSeconds: number) {
+      await wfCtxt.recv('a-topic', timeoutSeconds);
+      return;
+    }
+
   }
 
   test("workflow-sleep-oaoo", async () => {
     const workflowUUID = uuidv1();
     const initTime = Date.now();
     await expect(testRuntime.invokeWorkflow(WorkflowOAOO, workflowUUID).sleepWorkflow(2)).resolves.toBeFalsy();
-    expect(Date.now() - initTime).toBeGreaterThanOrEqual(1500);
+    expect(Date.now() - initTime).toBeGreaterThanOrEqual(2000);
```

Comment:
We picked a smaller number because the `setTimeout` function might be inaccurate. The actual sleep time could be slightly shorter than 2000 as well. So I'd recommend setting it to a larger number like `1950` but not `2000`.

Reply:
My understanding of the situation is that it may take bit longer but never less. I tried with 2010 and the test usually passes. With 2005 it always passes on my PC (I tried it 10 times in  row without a single failure).
But 1950 it is!